src/content/changelog/2019-10-07-api.md

```diff
@@ -9,19 +9,19 @@ version: 4.6.0
 
 ### Breaking
 
-* Removed `force` query parameter from [DELETE /object-storage/buckets/{clusterId}/{bucket}](https://developers.linode.com/api/v4/object-storage-buckets-cluster-id-bucket/#delete) endpoint.
+* Removed `force` query parameter from [DELETE /object-storage/buckets/{clusterId}/{bucket}](https://www.linode.com/docs/api/object-storage/#object-storage-bucket-remove) endpoint.
 
-* Removed `objects`, `size`, and `region` parameters from [GET /object-storage/buckets](https://developers.linode.com/api/v4/object-storage-buckets) and [GET /object-storage/buckets/{clusterId}/{bucket}](https://developers.linode.com/api/v4/object-storage-buckets-cluster-id-bucket) endpoints.
+* Removed `objects`, `size`, and `region` parameters from [GET /object-storage/buckets](https://www.linode.com/docs/api/object-storage/#object-storage-buckets-list) and [GET /object-storage/buckets/{clusterId}/{bucket}](https://www.linode.com/docs/api/object-storage/#object-storage-bucket-view) endpoints.
 
 ### Added
 
 * A `linode_reboot` Event action can now generate a `secondary_entity` object. A `secondary_entity` object displays information about an additional entity that is related to the Event.
-    * Use the Accounts collection to List Events [GET /account/events](/api/v4/account-events) or to View an Event [GET /account/events/{eventId}](/api/v4/account-events-event-id).
+    * Use the Accounts collection to List Events [GET /account/events](https://www.linode.com/docs/api/account/#events-list) or to View an Event [GET /account/events/{eventId}](https://www.linode.com/docs/api/account/#event-view).
 
-* The List Managed Stats endpoint [GET managed/stats](/api/v4/managed-stats) returns a list of Managed Stats on your Account in the form of x and y data points. You can use these data points to plot your own graph visualizations. These stats reflect the last 24 hours of combined usage across all managed Linodes on your account giving you a high-level snapshot of data for `cpu`, `disk`, `swap`, `network in`, and `network out`.
+* The List Managed Stats endpoint [GET managed/stats](https://www.linode.com/docs/api/managed/#managed-stats-list) returns a list of Managed Stats on your Account in the form of x and y data points. You can use these data points to plot your own graph visualizations. These stats reflect the last 24 hours of combined usage across all managed Linodes on your account giving you a high-level snapshot of data for `cpu`, `disk`, `swap`, `network in`, and `network out`.
 
 ### Fixed
 
-* Fixed a formatting error that prevented the [filtering and sorting](https://developers.linode.com/api/v4/#filtering-and-sorting) examples within the API documentation from correctly working as presented.
+* Fixed a formatting error that prevented the [filtering and sorting](https://www.linode.com/docs/api/) examples within the API documentation from correctly working as presented.
```

Comment:
can we make this link: http://www.linode.com/docs/api/#filtering-and-sorting

Reply:
yes! Thank you for reviewing...pushed those three things up.